### ISSUE TITLE
Occasional panel layout glitch: builder fields sometimes displayed at narrow width

### ISSUE THREAD

**Issue by gerrit** [TimOetting/kirby-builder#60]
Kirby 2.5.8, kirby-builder 1.1.0

When working with content in the panel, *sometimes* the fields in the builder region are rendered squished / at a narrow width. Haven't been able to reproduce consistently, but happens often for us. 

![bildschirmfoto 2018-10-28 um 22 54 21](https://user-images.githubusercontent.com/3675/47966892-92889000-e057-11e8-86a0-8dfc557a943f.png)

**Comment by francois-gm** [comment]
Figured out the glitch happens if you drag a bit the element before opening it. If I clicked on it without dragging it was fine. Then what I did was to force the css of the div to be 100% by adding !important as my guess is that when there's a drag event maybe properties gets reset or changed... It seems like it's okay now (have to test further).

In my builder snippets I've added the following before my html:
<style>.builder-entry-quickform-container{width:100%!important;}</style>
